cloud/amazon/ec2_eni.py

```diff
@@ -466,7 +516,7 @@ def main():
                             ],
                            required_if=([
                                ('state', 'present', ['subnet_id']),
-                               ('state', 'absent', ['eni_id']),
+                               ('state', 'absent', ['eni_id'])
```

Comment:
Ok. So I'm going to change attached to default to None, then I put the required_if line as you suggested. Sounds good?